pythermalcomfort/plots/matplotlib/ranges_tdb_v.py

```diff
@@ -67,9 +81,14 @@ def ranges_tdb_v(
         Axes to plot on. If None, a new figure and axes are created.
     legend : bool, default True
         Whether to add a default legend for the regions.
-    plot_kwargs : dict[str, Any] or None, optional
-        Additional keyword arguments forwarded to ``calc_plot_ranges`` for further
-        customization (e.g., cmap, band_colors, xlabel, ylabel, etc.).
+    **kwargs : Any
```

Comment:
I suggest we remove all the inputs like xlabel, ylabel, title, and figsize since they can be changed later on since the function returns the plot axis. Here we should only pass the necessary inputs that are needed to generated the figure. Kristina is looking into the Seaborn documentation to see how they implemented the functions there. Could you also please look into that as I mentioned a few weeks ago?